gpii/node_modules/lifecycleManager/src/LifecycleManager.js

```diff
@@ -736,6 +736,11 @@ var fluid = fluid || require("infusion"),
             options: handlerSpec.options
         }];
 
+        // Optionally include supportedSettings if there are any
```

Comment:
Please add to this comment an attribution to GPII-3119

Reply:
attributed